Example_Workflow/run.py

```diff
@@ -0,0 +1,131 @@
+import os
+import pymser
+import pandas as pd
+
+from modules.raspa_input import create_GCMC_input
+from modules.raspa_output import parse_GCMC
+
+# Ignore warnings
+import warnings
+warnings.filterwarnings("ignore")
+
+# Define the conditions of the simulation
+
+OutputPath = 'GCMC'
+FrameworkName = 'MgMOF-74'
+UnitCells = 'auto'
+NumberOfCycles = 1000
+UseChargesFromCIFFile = 'yes'
+GasComposition = {'CO2': 1.0}
+ExternalTemperature = 298.0
+ExternalPressure = 1e4
+DesiredProdCycles = 2000
```

Comment:
Instead of providing all the files in the GCMC folder, we can simply ask the user to enter the names of the folders and files they want to use.
```suggestion
OutputPath = 'GCMC'             # Please provide a directory
FrameworkName = 'MgMOF-74'      # Name of the CIF file
NumberOfCycles = 1000           # Number cycles to run per iteration
GasComposition = {'CO2': 1.0}   # Name of the DEF file
ExternalTemperature = 298.0     # Temperature
ExternalPressure = 1e4          # Pressure
DesiredProdCycles = 2000        # Number of desired production cyles
UseChargesFromCIFFile = 'yes'
UnitCells = 'auto'
```

Reply:
I've changed the code to enable the execution from the terminal, using the `argparse` package. Now every option is an argument of the `run.py` code.